src/importexport/musedata/internal/musedata.cpp

```diff
@@ -357,74 +357,75 @@ void MuseData::readNote(Part* part, QStringView s)
     QString dynamics;
     QStringView an = s.mid(31, 11);
     for (int i = 0; i < an.size(); ++i) {
-        if (an[i] == '(') {
```

Comment:
Such long chains of `else if`s become much more readable as a switch.
Particularly when several cases execute the same statement as here.

Reply:
That's to say: you're welcome to comment on the actual code changes (your first comment was indeed correct and on point), but don't we don't need code-review on everything else.